pop_paths_scripts/install_kernel_trinity_ltp.sh

```diff
@@ -1,3 +1,5 @@
+source vars.env
```

Comment:
im pretty sure this isnt the correct path since this is in the pop_paths_scripts folder. I also think you missed a few other files where tristan was using the .txt config style

Reply:
Yeah my bad, forgot to check host scripts. I think I got them all now.